.github/workflows/tail-call.yml

```
@@ -6,12 +6,14 @@ on:
       - 'Python/ceval.c'
       - 'Python/ceval_macros.h'
       - 'Python/generated_cases.c.h'
+      - '.github/workflows/tail-call.yml'
   push:
     paths:
       - 'Python/bytecodes.c'
       - 'Python/ceval.c'
       - 'Python/ceval_macros.h'
       - 'Python/generated_cases.c.h'
+      - '.github/workflows/tail-call.yml'
```

Little nit, sort like this (and the one above), otherwise CI changes look good 👍


```suggestion
      - '.github/workflows/tail-call.yml'
      - 'Python/bytecodes.c'
      - 'Python/ceval.c'
      - 'Python/ceval_macros.h'
      - 'Python/generated_cases.c.h'
```